server/routes/settings/radarr.ts

```diff
@@ -57,6 +58,10 @@ radarrRoutes.post<
         path: folder.path,
       })),
       tags,
+      urlBase:
+        req.body.baseUrl && req.body.baseUrl !== ''
```

Comment:
```suggestion
        req.body.baseUrl
```

Reply:
Thanks for pointing that out, I forgot about it.